mlir-sycl/include/mlir/Dialect/SYCL/Transforms/Passes.td

```diff
@@ -44,8 +45,7 @@ def SYCLMethodToSYCLCall : Pass<"sycl-method-to-sycl-call", "ModuleOp"> {
     Replace SYCLMethodOpInterface instances by SYCLCallOps.
   }];
   let constructor = "mlir::sycl::createSYCLMethodToSYCLCallPass()";
-  let dependentDialects =
-    ["memref::MemRefDialect", "LLVM::LLVMDialect", "polygeist::PolygeistDialect"];
```

Comment:
Thanks to @whitneywhtsang changes, we no longer need to depend on these dialects.